doc/index.rst

```diff
@@ -17,7 +17,8 @@ Please use the following citation in any publication using grid library:
     E. Vohringer‐Martinez, T. Verstraelen, P. W. Ayers, F. Heidar‐Zadeh
 
 
-The Grid source code is hosted on GitHub and is released under the GNU General Public License v3.0. We welcome
+The `Grid source code is hosted on GitHub <https://github.com/theochem/grid>`_ and is released under the
```

Comment:
I would prefer just "Grid source code" or "source code" to be highlighted, because for me usually short parts of a sentence are highlighted

Reply:
Fixed in a7dbd8d.